packages.txt

```diff
@@ -1172,30 +1358,33 @@ tcl8.5						install
 tcpd						install
 telnet						install
 tig						install
-time						install
+tk8.6-blt2.5					install
 tmux						install
 toilet						install
 toilet-fonts					install
 topgit						install
 tor						install
 tor-arm						install
+tor-geoipdb					install
```

Comment:
Probably due to package updates that haven't happened in `shell-etc` yet.
I haven't had time to do so yet, but I'm going to perform those upgrades (and merge the result back in `shell-etc`, then do `apt install nginx` then rebase your branch on top of that, so that should go away (and all the Debian boilerplate will be signed by an admin)